### PR TITLE
obmd data migration script

### PR DESCRIPTION
**Pull request by zenhack**
Per #928.

//cc @naved001

### CODE REVIEW

File: hil/commands/migrate_ipmi_info.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+"""Helper commands for moving IPMI info from the database to obmd.
+
+This is part of our obmd migration strategy; see the discussion at:
```
Comment (naved001) [inline hil/commands/migrate_ipmi_info.py]: forgot to link the discussion? #928

File: hil/commands/migrate_ipmi_info.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        # XXX: apparently we've incorrectly specified the relationship between
+        # Obms and nodes, such that the node attribute returns a list, even
+        # though there can only ever be one node. If we were keeping pluggable
+        # obm support for longer, We'd just fix this, but since we're removing
+        # the functionality soon it's easier to just do this and not have to
+        # worry about what other code we might disturb:
+        node = obm.node[0]
+
+        if obm.type != ipmi_api_name:
+            sys.exit(("Node %s{label} has an obm of unspported "
```
Comment (naved001) [inline hil/commands/migrate_ipmi_info.py]: why not skip it rather than exiting?

**Comment by xuhang57** [comment]
Not familiar with obmd. But overall, Igtm.

File: hil/commands/migrate_ipmi_info.py (hunk 2 above)
Comment (zenhack) [reply hil/commands/migrate_ipmi_info.py]: Given that we plan on ripping out tge driver mechanism entirely, that would mean potentially losing nodes; this is something an admin ought to know about.

File: hil/commands/migrate_ipmi_info.py (hunk 1 above)
Comment (zenhack) [reply hil/commands/migrate_ipmi_info.py]: Yep, good spot. Will fix.

**Comment by Izhmash** [comment]
How is the rest of the migration strategy going to be implemented? From issue #928 I see:

>  4. The final migration-script(2) will delete the old obm tables/columns, and make the OBMd stuff not nullable. 

Will this cleanup happen in another file?

**Comment by zenhack** [comment]
@Izhmash, yeah, that would be a migration script added in `hil/migrations/versions/`

**Comment by zenhack** [comment]
...perhaps we need to plan to tag a version between this pr and the one that introduces the migration script? The strategy involves an intermediate upgrade, so we should be able to refer to the specific intermediate version.

**Comment by Izhmash** [comment]
@zenhack I think that sounds like a good idea, yeah. Would definitely help relieve user confusion. What would be the new version number?

**Comment by zenhack** [comment]
@Izhmash, 0.3 is next on the list. We've got a [milestone][1] for that; maybe we should add the second part of this to the 0.4 milestone, and then start focusing on triaging the 0.3 issues so we can get that out the door.

[1]: https://github.com/CCI-MOC/hil/milestone/9

**Pull request merged**